modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/maven/MavenDeployableProjectModelBuilder.java

```diff
@@ -128,11 +133,15 @@ protected DeployableProjectModel doBuild(Model pomModel, ArtifactCoordinates dep
     // Get exported resources and packages
     try {
       List<String> packages = getAvailablePackages(pomModel.getBuild());
-      List<String> resources = getAvailableResources(pomModel.getBuild());
+      List<String> muleResources = getAvailableMuleResources(pomModel.getBuild());
+      List<String> nonMuleResources = getAvailableNonMuleResources(pomModel.getBuild());
```

Comment:
is there something else you do with this other than getting the configs? why not call `getConfigs` here directly instead of propagating everything needed to calculate it later?

Reply:
We could avoid calculating the configs if it's not necessary because they're already present in the `mule-artifact.json`, but I can change it since it shouldn't be an expensive task.